addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -64,7 +71,10 @@
 		<description>Control the zone</description>
 		<channels>
 			<channel id="power" typeId="power" />
+			<!--
+            We will provide an input channel dynamically. 
```

Comment:
@kaikreuzer recently stated that dynamic channels are not the way forward, is there a reason to use them here?